plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -156,7 +155,7 @@ private void setupDataBinding() {
     bindingContext = new DataBindingContext();
```

Comment:
Do we need to pass in the DBC when it's accessible from the `bindingContext` field?